setup.sh

```diff
@@ -84,7 +84,7 @@ if [[ -n "${CI-}" ]]; then # skip cache bootstrap for CI
     echo "Skipping install since CI is defined"
 else
     which bundle || gem install bundler
-    bundle check || bundle install || echo "⚠️  Skipping bundle install since no or an old ruby version is installed"
+    bundle check || bundle install || echo "⚠️  Skipping bundle install, please check if you have the required ruby version installed"
```

Comment:
@caldrian I am not sure we should just skip it, I'd rather like that the one executing the script match the requirements

Reply:
I don't agree because the gems are not needed for anything locally.
Only for devs who like to execute fastlane and other dependencies need it.